pkg/gb/gameboy.go

```diff
@@ -50,9 +50,11 @@ type Gameboy struct {
 	interruptsOn       bool
 	halted             bool
 
-	cbInst    []func()
+	cbInst    [0x100]func()
 	InputMask byte
 
+	mainInst [0x100]func()
```

Comment:
Could you put the `mainInst` above the `cbInst` so that they are grouped together?